maven/features/org.jboss.tools.maven.test.feature/feature.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<feature id="org.jboss.tools.maven.test.feature" label="%featureName" version="1.9.100.qualifier" provider-name="%featureProvider"
+<feature id="org.jboss.tools.maven.test.feature" label="%featureName" version="1.9.300.qualifier" provider-name="%featureProvider"
```

Comment:
veresion should be 0.0.0 to ensure the latest plugin version is fetched and then not necessary to change versions for each patch.